js/client/modules/@arangodb/testutils/testing.js

```diff
@@ -164,8 +164,9 @@ let optionsDocumentation = [
 
 const isSan = (
   global.ARANGODB_CLIENT_VERSION(true).asan === 'true' ||
-    global.ARANGODB_CLIENT_VERSION(true).tsan === 'true');
-
+  global.ARANGODB_CLIENT_VERSION(true).tsan === 'true' ||
+  internal.coverage
+);
```

Comment:
The name `isSan` is now misleading, maybe change it to something more accurate?

Reply:
sugestions?